roseau/load_flow/network/electrical_network.py

```diff
@@ -56,27 +56,31 @@ class ElectricalNetwork:
     #
     def __init__(
         self,
-        buses: Union[list[Bus], dict[Any, Bus]],
-        branches: Union[list[AbstractBranch], dict[Any, AbstractBranch]],
-        loads: Union[list[AbstractLoad], dict[Any, AbstractLoad]],
-        voltage_sources: Union[list[VoltageSource], dict[Any, VoltageSource]],
+        buses: Union[list[Bus], dict[Id, Bus]],
+        branches: Union[list[AbstractBranch], dict[Id, AbstractBranch]],
+        loads: Union[list[AbstractLoad], dict[Id, AbstractLoad]],
+        voltage_sources: Union[list[VoltageSource], dict[Id, VoltageSource]],
         special_elements: list[Element],
```

Comment:
In a next PR, this "special_elements" must be deleted. I think it is the last one.